utils.js

```diff
@@ -728,69 +728,51 @@ function metaContentType(options) {
 //   ];
 // }
 
-
-const defaultSources = new Map([["audio", new Map([["src", {
-  type: srcType
-}]])], ["embed", new Map([["src", {
-  type: srcType
-}]])], ["img", new Map([["src", {
-  type: srcType
-}], ["srcset", {
-  type: srcsetType
-}]])], ["input", new Map([["src", {
-  type: srcType
-}]])], ["link", new Map([["href", {
-  type: srcType,
-  filter: linkUnionFilter
-}], ["imagesrcset", {
-  type: srcsetType,
-  filter: linkHrefFilter
-}]])], ["meta", new Map([["content", {
-  type: metaContentType,
-  filter: metaContentFilter
-}]])], ["object", new Map([["data", {
-  type: srcType
-}]])], ["script", new Map([["src", {
-  type: srcType,
-  filter: scriptSrcFilter
-}], // Using href with <script> is described here: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/script
-["href", {
-  type: srcType,
-  filter: scriptSrcFilter
-}], ["xlink:href", {
-  type: srcType,
-  filter: scriptSrcFilter
-}]])], ["source", new Map([["src", {
-  type: srcType
-}], ["srcset", {
-  type: srcsetType
-}]])], ["track", new Map([["src", {
-  type: srcType
-}]])], ["video", new Map([["poster", {
-  type: srcType
-}], ["src", {
-  type: srcType
-}]])], // SVG
-["image", new Map([["xlink:href", {
-  type: srcType
-}], ["href", {
-  type: srcType
-}]])], ["use", new Map([["xlink:href", {
-  type: srcType
-}], ["href", {
-  type: srcType
-}]])] // [
-//   'webpack-import',
-//   new Map([
-//     [
-//       'src',
-//       {
-//         type: webpackImportType,
-//       },
-//     ],
-//   ]),
-// ],
-]);
+// defaultSources is populated in addDataSources
```

Comment:
`utils.js` is copy from `html-loader`. If this file is changed, we will be hard to synchronize with `html-loader` update.